### PR TITLE
Fix ZipFile version number, consolidate project file

### PR DESCRIPTION
**Pull request by mellinoe**
The version number in this assembly was not being set, so it was defaulting to 999.999.999.999. We are overriding the version number in our internal build that wraps this project, so this wasn't an issue for our NuGet packages. I've also consolidated the facade project for net46 into the single root project.

### CODE REVIEW

**Comment by stephentoub** [comment]
cc: @weshaggard

**Comment by mellinoe** [comment]
@dotnet-bot Test Innerloop Ubuntu Release Build and Test

(Error cloning the repo)

File: src/System.IO.Compression.ZipFile/src/project.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+        "System.IO.FileSystem.Primitives": "4.0.0",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.0.10",
+        "System.Text.Encoding": "4.0.10"
+      }
+    },
+    "net46": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.0-rc2-23530"
```
Comment (weshaggard) [inline src/System.IO.Compression.ZipFile/src/project.json]: Please go ahead and remove the prerelease id from this.

**Comment by weshaggard** [comment]
LGTM

**Comment by mellinoe** [comment]
@dotnet-bot Test Innerloop Windows_NT Debug Build and Test
https://github.com/dotnet/corefx/issues/5185

**Pull request merged**